include/cctz/civil_time_detail.h

```diff
@@ -566,6 +567,198 @@ CONSTEXPR_F civil_day prev_weekday(civil_day cd, weekday wd) noexcept {
   }
 }
 
+////////////////////////////////////////////////////////////////////////
+
+enum class weekstart
+{
+  monday_zero,
+  monday_one,
+  sunday_zero,
+  sunday_one,
+};
+
+CONSTEXPR_F int weekday_to_number(weekday wd, weekstart ws) noexcept {
```

Comment:
Other possiblity could be to have

- weekday_to_sunday_zero
- weekday_to_sunday_one
- weekday_to_monday_zero
- weekday_to_monday_one

This use using there would be shorter and we would not need enum. Compare:
auto wd_num = cctz::weekday_to_number(1, cctz::weekstart::monday_zero);
auto wd_num = cctz::weekday_to_monday_zero(1);

Reply:
> We definitely need those cast as we are example converting them for external APIs.

> Other possiblity could be to have
> 
> * weekday_to_sunday_zero
> * weekday_to_sunday_one
> * weekday_to_monday_zero
> * weekday_to_monday_one

In addition to @derekmauro's comment, I would also suggest that such conversion functions belong in the converting code (where the "from" and "to" are clear, just like the existing `tm` case), rather than having N public helper functions in CCTZ.